README.md

```diff
@@ -12,19 +12,22 @@
 3. Rename the copy to `.env`.
 4. Replace all the fields within '<>' by actual values (not keeping the <>). They can be any of your choosing.
 
-   > :bulb: Note that you don't have to create any account or credentials prior to this setup. It will be done automatically when you first run the containers.
+   > 💡 Note that you don't have to create any account or credentials prior to this setup. It will be done automatically when you first run the containers.
 
 5. Open a terminal window in the root directory of the repository (UInnovate).
 6. Run `docker compose --profile tool up` to start the containers.  
    You can stop them at any time by running `docker compose --profile tool stop`.
-   
-    > :bulb: Note that the pgadmin container will only be targeted by a docker compose command if you specify `--profile tool` as shown above. Otherwise, only the PostgreSQL and the PostgREST containers will be affected. Leaving it out can become handy if you try to reset the database, but you don't want to lose your server connection on pgadmin!  
-
-7. In a web browser window, access to localhost:5050
-8. Log in to pgAdmin with the credentials you provided in your `.env` file.
-9. On the home page, click on "Add New Server".
-10. In the "General" tab, enter a Name for your local PostgreSQL server (e.g. "UInnovate Local PostgreSQL").
-11. In the "Connection" tab, fill the fields with the following values :
+
+   > 💡 Note that the pgadmin container will only be targeted by a docker compose command if you specify `--profile tool` as shown above. Otherwise, only the PostgreSQL and the PostgREST containers will be affected. Leaving it out can become handy if you try to reset the database, but you don't want to lose your server connection on pgadmin!
+
+7. Run `.\refresh_database.sh` in your terminal to populate the db. (Note: \ on Windows, / on Linux)
+   > 💡 Note: You may need to shut down the db and server containers and restart them after running the refresh database command. To do so, use `docker compose down && docker compose up  -d` in your terminal after the .\refresh_database.sh command
```

Comment:
I'm not sure this is accurate, as `docker compose down` would delete the containers. So, on `docker compose up`, the containers would be re-created from scratch, and the db wouldn't have any data in it. I suggest using `docker compose stop` instead of `docker compose down`:
```suggestion
   > 💡 Note: You may need to shut down the db and server containers and restart them after running the refresh database command. To do so, use `docker compose stop && docker compose up  -d` in your terminal after the .\refresh_database.sh command
```

Reply:
Okay, as long as you tested it like that and it was great that makes sense and is good with me! :)